charts/sda-svc/templates/s3-inbox-secrets.yaml

```diff
@@ -7,11 +7,10 @@ metadata:
   name: {{ template "sda.fullname" . }}-inbox
 type: Opaque
 data:
-  dbPassword: {{ include "dbPassInbox" . | b64enc }}
-  dbUser: {{ include "dbUserInbox" . | b64enc }}
-  mqPassword: {{ include "mqPassInbox" . | b64enc }}
-  s3InboxAccessKey: {{ .Values.global.inbox.s3AccessKey | b64enc }}
-  s3InboxSecretKey: {{ .Values.global.inbox.s3SecretKey | b64enc }}
+  dbPassword: {{ required "DB password is required" (include "dbPassInbox" .) | quote | trimall "\"" | b64enc }}
```

Comment:
Maybe it's obvious, but why do we require `quote | trimall "\""` in this case?

Reply:
The DB password can contain characters  that causes the string to quoted, and we don't want to keep the quotation characters when we quote it before base64 encoding it. At least that's from the old helm versions, seems like a perfect thing to create a test for.